Tests/Modules/EventsModuleSpecs.swift

```diff
@@ -239,11 +239,16 @@ class EventsModuleSpecs: QuickSpec {
                                 case let .success(event):
                                     expect(event).toNot(beNil())
                                     expect(event).to(beAKindOf(Event.self))
-                                    expect(event.id).to(equal("f82c3ba03e41f7e8a7608363cc6c0390183c3f83"))
-                                    expect(event.createdBy?.id).to(equal("11111"))
-                                    expect(event.eventType).to(equal(.itemCreated))
-                                    expect(event.sessionId).to(equal("70090280850c8d2a1933c1"))
-
+                                    expect(event.id).to(equal("1a4ade15-b1ff-4cc3-89a8-955e1522557c"))
+                                    expect(event.createdBy?.id).to(equal("55555"))
+                                    expect(event.sessionId).to(beNil())
+                                    switch event.source?.itemValue {
+                                    case let .file(file):
```

Comment:
item type is probably okay because we used it to deserialize to file but maybe an assertion just to confirm the type?